packages/css/src/components/Form/TextArea.stories.js

```diff
@@ -0,0 +1,92 @@
+export default {
+  title: 'Components/Forms/TextArea',
+  argTypes: {
+    label: { control: 'text' },
+    size: {
+      control: { type: 'select' },
+      options: ['small', 'medium', 'large'],
+    },
+    state: {
+      control: { type: 'select' },
+      options: ['default', 'disabled', 'active', 'error', 'success'],
```

Comment:
What does the `active` state do? Is `default` and `active` the same thing?

Reply:
Leftover.